kivy/logger.py

```diff
@@ -302,14 +397,18 @@ def isatty(self):
 
 
 def logger_config_update(section, key, value):
-    if LOG_LEVELS.get(value) is None:
-        raise AttributeError('Loglevel {0!r} doesn\'t exists'.format(value))
-    Logger.setLevel(level=LOG_LEVELS.get(value))
+    if KIVY_LOG_MODE != "PYTHON":
+        if LOG_LEVELS.get(value) is None:
+            raise AttributeError('Loglevel {0!r} doesn\'t exists'.format(value))
+            Logger.setLevel(level=LOG_LEVELS.get(value))
```

Comment:
Should not look like the following?

```suggestion
    if KIVY_LOG_MODE != "PYTHON":
        if LOG_LEVELS.get(value) is None:
            raise AttributeError('Loglevel {0!r} doesn\'t exists'.format(value))
        Logger.setLevel(level=LOG_LEVELS.get(value))
```

Reply:
Thank you!